modules/default-requirements-secured-cluster-services.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-default-requirements.adoc
+:_content-type: CONCEPT
+[id="default-requirements-secured-cluster-services_{context}"]
+= Secured cluster services
+
+Secured cluster services contain the following components:
+
+* Sensor
+* Admission controller
+* Collector
+
+[id="default-requirements-secured-cluster-services-sensor_{context}"]
+== Sensor
+
+Sensor monitors your Kubernetes and OpenShift Container Platform clusters. These services currently deploy in a single deployment, which handles interactions with the Kubernetes API and coordinates with Collector.
+
+[discrete]
+== Memory and CPU requirements
+
+The following table lists the minimum memory and storage values required to install and run Central.
+
+|===
+| Sensor | CPU | Memory
+
+| *Request*
+| 2 cores
+| 4 GiB
+
+| *Limit*
+| 4 cores
+| 8 GiB
+|===
+
+[id="default-requirements-secured-cluster-services-admission-controller_{context}"]
+== Admission controller
+
+The Admission controller prevents users from creating workloads that violate policies you configure.
+
+[discrete]
+== Memory and CPU requirements
+
+By default, the admission control service runs 3 replicas. The following table lists the request and limits for each replica.
+
+|===
+| Admission controller | CPU | Memory
+
+| *Request*
+| .05 cores
+| 100 MiB
+
+| *Limit*
+| .5 cores
+| 500 MiB
+|===
+
+[id="default-requirements-secured-cluster-services-collector_{context}"]
+== Collector
+
+Collector monitors runtime activity on each node in your secured clusters. It connects to Sensor to report this information. The collector pod has three containers. The first container is collector, which actually monitors and reports the runtime activity on the node. The other two are compliance and node-scanner.
+
+[CAUTION]
+====
+To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
+====
+
+[discrete]
+== Memory and CPU requirements
+
+By default, the admission control service runs 3 replicas. The following table lists the request and limits for each replica.
+
+|===
+| Collector | | CPU | Memory
+
+.2+| *Collector Container*
+| *Request*
+| .05 cores
+| 320 MiB
+
+| *Limit*
+| .75 cores
+| 1000 MiB
+
+.2+| *Compliance Container*
+| *Request*
+| .01 cores
+| 10 MiB
+
+| *Limit*
+| 1 core
+| 2000 MiB
+
+.2+| *Node-scanner Container*
+| *Request*
+| .01 cores
+| 10 MiB
+
+| *Limit*
+| .75 cores
+| 2000 MiB
```

Comment:
```suggestion
| 1 cores
| 500 MiB
```

This is the current state. The memory limit should stay, but the CPU may be lowered (confirmation pending).